config.js

```diff
@@ -20,7 +20,7 @@ const validate = (config) =>
 const env = { local, development, production }
 const config = {
   ...env.production,
-  ...env[NODE_ENV],
+  // ...env[NODE_ENV],
```

Comment:
Is this a mistake?

Reply:
yeah I was trying to avoid the bug discussed on slack